language/move-vm/runtime/src/loader.rs

```diff
@@ -612,51 +617,46 @@ impl Loader {
     pub(crate) fn verify_module_for_publication(
         &self,
         module: &CompiledModule,
+        bundle_verified: &BTreeMap<ModuleId, CompiledModule>,
```

Comment:
- Would you mind adding some comments to explain the meaning of these `bundle_verified` and `bundle_unverified` params in each procedure + their relationship to `module`?
- It makes me a big nervous to have this be `pub(crate)` if its correctness depends on supplying the right `bundle_verified` or `bundle_unverified`... Is there a way that we can make this API private and expose a public one without these parameters?

Reply:
Both addressed, the function is now a private function and the `pub(crate)` function is `verify_module_bundle_for_publication` which does not include these two variables.